### PR TITLE
Check hardware watchdog device

### PR DESCRIPTION
**Pull request by OniriCorpe**
cf: https://github.com/YunoHost-Apps/watchdog_ynh/issues/2

### CODE REVIEW

**Comment by OniriCorpe** [comment]
an alternative would be to show an PRE_INSTALL.md

File: scripts/install [hunk 1]
```diff
@@ -8,6 +8,16 @@
 source _common.sh
 source /usr/share/yunohost/helpers
 
+#=================================================
+# WATCHDOG DEVICE CHECK
+#=================================================
+
+ynh_script_progression --message="Checking the hardware watchdog device presence..."
+
+if [ ! -f /dev/watchdog ]; then
```
Comment (tituspijean) [inline scripts/install:17]: I think that device is only created after installing the appropriate package. 🫠

**Review by ericgaspar — APPROVED** [review]

File: scripts/install (hunk 1 above)
Comment (tituspijean) [reply scripts/install:17]: erf... I had that comment pending for months, sorry :(

Comment (domrc) [reply scripts/install:17]: Hi, sorry to bump this pull request, but that `-f` is stopping this from installing on my device that has a proper `/dev/watchdog` and `/dev/watchdog0`. The problem is that `-f` is used to check if the file `/dev/watchdog` exists *and* is a regular file, which it is not.
Flags that would work would be `-e`, which only checks if the file exists, or `-c`, which checks if it exists and is a special character device

**Pull request merged**